tests-build/tests/fail/macros_invalid_input.stderr

```diff
@@ -1,113 +1,113 @@
 error: the `async` keyword is missing from the function declaration
- --> $DIR/macros_invalid_input.rs:6:1
+ --> tests/fail/macros_invalid_input.rs:6:1
```

Comment:
Probably not a big deal, but do you know where these changes came from?

Reply:
It seems that the changes came from `trybuild` version update. I modified the error text like [this line](https://github.com/tokio-rs/tokio/pull/6593/files#diff-3778ff38bebb2786ca2b45f4704af832e54cd75369b7b9215eae010cddc58a09R363) and updated reference compiler output with `TRYBUILD=overwrite`, then every `$DIR` changed. Same changes appear in other recent PRs like #5643.

Reply:
> Probably not a big deal, but do you know where these changes came from?

It seems that the changes came from `trybuild` version update. I modified the error text like [this line](https://github.com/tokio-rs/tokio/pull/6593/files#diff-3778ff38bebb2786ca2b45f4704af832e54cd75369b7b9215eae010cddc58a09R363) and updated reference compiler output with `TRYBUILD=overwrite`, then every `$DIR` changed. Same changes appear in other recent PRs like #5643.